### DISCUSSION TITLE
Coolify Cloud not up to date

### DISCUSSION THREAD

**Discussion by wilfredinni** [coollabsio/coolify#3531] [General]
Is coolify Cloud not up to date with the GH releases? I want to try a service added in one of the latest, but im stuck in v. 323

Thank you

**Comment by peaklabs-dev** [comment]
This is intended as we made some big changes and have to ensure stability. You can read more here https://github.com/coollabsio/coolify/blob/main/RELEASE.md. i have a PR open that improves the release guide even more to make it more clear.
